.pre-commit-config.yaml

```diff
@@ -3,4 +3,6 @@ repos:
     rev: stable
     hooks:
     - id: black
-      args: [--check]
\ No newline at end of file
+      args: [--check]
+    - id: pyupgrade
+      args: [--py37-plus]
```

Comment:
Add one and only one carriage return to the end of the file so GitHub removes the ⛔ .

Reply:
complete :smile: